templates/image-builder.yml

```diff
@@ -130,6 +130,8 @@ objects:
                 key: proxy
                 name: recommendations-secrets
                 optional: true
+          - name: CHANNEL
+            value: ${CHANNEL}
         volumeMounts:
           - name: config-volume
             mountPath: /app/config
```

Comment:
This is missing the variable definition further down the file. But if the intention was not to set it explicitly it is fine.

Reply:
The definition should come implicitly from 
https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/products/osdv4/environments/stage.yml?ref_type=heads#L18
by stating that this template is "staging"
https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/insights/image-builder/deploy.yml?ref_type=heads#L78
for example